ci/test_supported_plugins.rb

```diff
@@ -0,0 +1,253 @@
+# encoding: utf-8
+
+# Test the 
+# - build (rspec), 
+# - packaging (gem build)
+# - and deploy (bin/logstash-plugin install)
+# of a plugins inside the current Logstash, using its JRuby
+# Usage example:
+# bin/ruby ci/test_supported_plugins.rb -p logstash-integration-jdbc
+# bin/ruby ci/test_supported_plugins.rb -t tier1 -k input,codec,integration
```

Comment:
🤔 the usage is missing a note about needing a clean bootstrap + `installDevelopmentGems`.

Reply:
we need the Logstash JRuby to be present, so at least here we should execute `./gradlew assemble`, to setup the environment to be able to run the Ruby script.

We can split this in 2:
- in the shell script invoke the minimal Gradle task that setup the environment to be able to run the Ruby script
- in the Ruby script executes all the steps that are fundamental to have Logstash in right posture to execute the tests; which means executing the Gradle's `installDevelopmentGems` tasks

Reply:
handled with b30ad380adb9cf70a039c2752fadb44625a0865b